pureconfig/README.md

```diff
@@ -70,6 +70,7 @@ rabbitConfig {
 myConfig {
   hosts = ["localhost:5672"]
   virtualHost = "/"
+  addressResolverType = "Default" // other possible values: List, DnsRecord, DnsSrvRecord
```

Comment:
Not sure if this belongs to readme, not all values should be here.

Reply:
So where do you recommend to place the list of possible values?

Reply:
Since we have case-classes as the main configuration way in projects, it's them who you can explore for possible values, not any `reference.conf` or such file. It's not ideal, I agree, but having two places would cause their desync sooner or later.